cobrand/views.py

```diff
@@ -32,6 +32,16 @@ class CobrandCompanyCreateView(django.views.generic.edit.CreateView):
     template_name = 'cobrand/create.html'
     form_class = forms.CobrandCompanyCreateForm
 
+    def post(self, request, *args, **kwargs):
+        company_name = request.POST.get('company_name')
+        if company_name:
+            try:
+                cobrand_company = models.CobrandCompany.objects.get(company_name=company_name)
```

Comment:
@waseem-omar I think perhaps an ``exists()`` query is both a little simpler and a little more understandable here. If you do it with ``exists()``, you just have an ``if ...`` statement without a surrounding ``try ... except`` block